exonum/src/runtime/dispatcher/mod.rs

```diff
@@ -44,13 +47,68 @@ mod tests;
 struct ServiceInfo {
     runtime_id: u32,
     name: String,
+    status: InstanceStatus,
+}
+
+/// Lookup table for the committed service instances.
+#[derive(Debug, Default)]
+struct CommittedServices {
+    instances: HashMap<InstanceId, ServiceInfo>,
+    instance_names: HashMap<String, InstanceId>,
+}
+
+impl CommittedServices {
+    fn insert(&mut self, id: InstanceId, info: ServiceInfo) {
+        let name = info.name.clone();
+        self.instances.insert(id, info);
+        self.instance_names.insert(name, id);
+    }
+
+    fn get_runtime_for_active_instance(&self, id: InstanceId) -> Option<u32> {
```

Comment:
To be more clear, I'd propose to rename it to `get_runtime_id_for_active_instance` ?